src/probnum/diffeq/steprule.py

```diff
@@ -1,28 +1,18 @@
-"""
-"""
-
 from abc import ABC, abstractmethod
 
-__all__ = ["StepRule", "ConstantSteps", "AdaptiveSteps"]
-
 
 class StepRule(ABC):
-    """
-    (Adaptive) step size rules for ODE solvers.
-    """
+    """(Adaptive) step size rules for ODE solvers."""
 
     @abstractmethod
     def suggest(self, laststep, errorest, **kwargs):
-        """
-        Suggests a new step h_{n+1} given error estimate
-        e_n at step h_n.
-        """
+        """Suggest a new step h_{n+1} given error estimate e_n at step h_n."""
         raise NotImplementedError
 
     @abstractmethod
     def is_accepted(self, proposedstep, errorest, **kwargs):
-        """
-        Checks if proposed step should be accepted or not.
+        """Check if the proposed step should be accepted or not.
```

Comment:
Shouldn't it be either `"""Check if ..."""` or `"""\nCheck if ...\n"""`?

Reply:
Does not have to: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings "The summary line may be on the same line as the opening quotes or on the next line."

But I'm fine with changing that of course :)

Reply:
I agree! My intention behind this comment was rather that the rest of the repository does it in the style I proposed here, so it was more of a consistency thing instead of a right/wrong thing :)